inst/qualification/qualification_map.R

```diff
@@ -41,7 +41,7 @@ qualification_map <- function(
   })
 
   ### Read in specs spreadsheet
-  spec <- read.csv(system.file("qualification", "qualification_specs.csv", package = "gsm"))
+  spec <- read.csv(system.file("qualification", "qualification_specs.csv", package = "gsm.qc"))
```

Comment:
@lauramaxwell I believe this was causing some funkiness in the push to dev earlier this morning